google/cloud/forseti/common/gcp_api/container.py

```diff
@@ -182,8 +238,14 @@ def get_serverconfig(self, project_id, zone):
         """
 
         try:
-            return self.repository.projects_zones.get_serverconfig(project_id,
-                                                                   zone)
+            if zone and not location:
+                return self.repository.projects_zones.get_serverconfig(
+                    project_id, zone=zone)
+            elif location:
```

Comment:
nit: purely a matter of personal taste, it would be nicer if the `location` condition is first to make its precedence more clear; the condition also seems simpler to boot.
```
if location:
   do this
elif zone:
   do that
```

Reply:
Done.